source/iface/sub_group.rst

```diff
@@ -4,6 +4,145 @@
 
 .. _sub_group:
 
-*******************
+===================
 ``sycl::sub_group``
-*******************
+===================
+
+::
+
+   class sub_group;
+
+The ``sub_group`` class encapsulates all functionality required to
+represent a particular sub-group within a parallel execution.
+It is not user-constructible.
+
+.. warning::
```

Comment:
Now I'm not really sure if we need this warning exactly on this page, but anyway, we'll fix it if needed.